rbc/tests/heavydb/test_text_encoding_dict.py

```diff
@@ -75,28 +75,28 @@ def test_getstringid_from_arg(x, m, text, y):
             y[0] = x.string_dict_proxy.getStringId(text)
             return 1
 
-        @heavydb('int32(TableFunctionManager, OutputColumn<TextEncodingDict> | input_id=args<>)',  # noqa: E501
-                 devices=['cpu'])
+        # Using TableFunctionManager argument implies devices=['cpu']
+        @heavydb('int32(TableFunctionManager, OutputColumn<TextEncodingDict> | input_id=args<>)')  # noqa: E501
```

Comment:
I thought that compiling this UDTF for GPU would result in a compilation failure, as the external functions used externally are declared with `|CPU` device suffix.

Reply:
It failed badly in `External::match_signature` because the external functions were not available for the `GPU` target while now the GPU target will be skipped in `match_signature` so that the compilation for `CPU` could finish.